tap_trello/discover.py

```diff
@@ -40,6 +40,12 @@ def do_discover():
         for field_name in stream.replication_keys:
             metadata.write(mdata, ('properties', field_name), 'inclusion', 'automatic')
 
+        # mark unsupported fields as 'unsupported' in the metadata
+        for field_name in stream.unsupported_fields:
+            metadata.write(mdata, ('properties', field_name), 'inclusion', 'unsupported')
+            metadata.write(mdata, ('properties', field_name), 'unsupported-description', \
```

Comment:
As per this [code](https://github.com/singer-io/tap-salesforce/blob/master/tap_salesforce/__init__.py#L233), we added description of unsupported fields.